post-project.cmake

```diff
@@ -47,8 +47,36 @@ cmake_dependent_option(
   BUILD_DOCUMENTATION
   OFF
 )
-option(INSTALL_GENERATED_HEADERS "Generate and install standard headers" ON)
-option(INSTALL_PKG_CONFIG_FILE "Generate and install standard .pc file" ON)
+
+option(BUILD_PYTHON_INTERFACE "Build the python bindings" ON)
+cmake_dependent_option(
+  BUILD_ONLY_PYTHON_INTERFACE
```

Comment:
As discussed in another thread, I would suggest renaming these variables to `BUILD_PYTHON_INTERFACE_ONLY`. The reason is that the current name is slightly awkward.

Reply:
Thanks for raising this. We discussed this and `BUILD_STANDALONE_PYTHON_INTERFACE` seems more relevant. Would you agree with this naming @cmastalli ?

Reply:
I would!